apps/dispatch-service/src/app/submission/complete.processor.ts

```diff
@@ -253,7 +257,7 @@ export class CompleteProcessor extends WorkerHost {
         returnValue: flow.job.returnvalue.data,
         required: flow.job.data.required,
         description: flow.job.data.description,
-        errorMessage,
+        errorMessage: flow.job.data.errorMessage,
```

Comment:
errorMessage local var already was prefixed with `flow.job.data.errorMessage` (see line 245).  This fix seems to just remove the extra info from `flow.job.data.moreInfo` (line 247) or `flow.job.data.validator` (line 250).

do you really want to remove info here?

Reply:
Not at all: this seems to have gotten removed on accident. Thanks!

Reply:
I see more of what happened here. `errorMessage` was indeed already locally defined.